exercises/operations/felt02.cairo

```diff
@@ -0,0 +1,31 @@
+%lang starknet
+
+# Felts supports basic math operations.
+# High level syntax allows one-line multiple operations while low level syntax doesn't.
+# Exercice source: https://www.cairo-lang.org/docs/how_cairo_works/cairo_intro.html#field-elements
+
+# I AM NOT DONE
+
+# TODO
+# Write this function body in a high level syntax
+func poly_high_level(x : felt) -> (res : felt):
+    # return x³ + 23x² + 45x + 67 according to x
+    return (res=res)  # Do not change
+end
+
+# TODO
+# Write this function body in a low level syntax (result must be stored in [ap - 1] before ret)
+func poly_low_level(x : felt):
+    # return x³ + 23x² + 45x + 67 according to x
+    ret  # Do not change
+end
```

Comment:
I don't think we need any low level syntax here.
This section is about felt as a type. Not the syntax.

Reply:
Especially since this is one of the first exercises in the course. At this point, the student is not even aware of the low level syntax. DOn't mention low/high level syntax and just ask for the arithmetic operations

Reply:
I agree, please see my note above :arrow_up:

Reply:
@tdelabro, do you disagree with the proposition about the theme splitted across the synoptic?

Reply:
I did changes according to your feedback and my initial note.

Here is the new synoptic (`<--` shows new exercises):

```
exercises/
    syntax/
      - syntax01
      - syntax02
      - syntax03
      - syntax04
      - syntax05
    operations/
      - operations00 <--
    strings/
      - strings00
      - strings01
    storage/
      - storage01
      - storage02
      - storage03
    implicit_arguments/
      - implicit_arguments01
      - implicit_arguments02
      - implicit_arguments03
    recursions/
      - recursion01
      - array01
      - array02
      - array03
      - array04
      - struct01
      - collatz_sequence
    registers/
      - registers00
      - registers01
      - registers02
      - registers03
      - registers04 <--
    revoked_references/
      - revoked_references01
    builtins/
      - bitwise
    tricks/
      - no_conditionals
      - assert_bool
      - inline_if
    hints/
      - hints00
      - hints01```